LFPy/cell.py

```diff
@@ -21,7 +21,6 @@
 import numpy as np
 import scipy
 import sys
-import posixpath
```

Comment:
Here it seems like a number of changes will revert recent changes to LFPy:master since you started your modifications on an even older version. Just for the future, this is why you should create a branch. So revert this changeset here, and where I've marked below.